core/src/main/java/org/elasticsearch/cluster/routing/allocation/AllocationService.java

```diff
@@ -143,7 +176,7 @@ public AllocationService(Settings settings, AllocationDeciders allocationDecider
      * If the same instance of the routing table is returned, then no change has been made.
      */
     public RoutingAllocation.Result reroute(ClusterState clusterState) {
-        return reroute(clusterState, false);
+        return reroute(clusterState, "reroute");
```

Comment:
That's what I was going to do. I would not have done this change manually. ;) 

I was more concerned about the increase in the changeset just for this signature change.